bin/Cargo.toml

```diff
@@ -58,3 +59,4 @@ default = []
 # DO NOT REMOVE!!
 custom-protocol = ["tauri/custom-protocol"]
 debug = ["tauri/devtools"]
+stacks = ["ethui-stacks"]
```

Comment:
I believe the syntax now is `"dep:ethui-stacks"` to make it clear it's a dependency and not another feature flag